src/index.js

```diff
@@ -131,8 +131,8 @@ class Service {
 
     let { query, options } = multiOptions(id, params, this.id);
 
-    // We can not update the id
-    delete data[this.id];
```

Comment:
Isn't the idea with `PUT` that should be sending the whole resource (including the id)? If we add the custom `id` back into the `data` object then you can make a request like so:

```json
// PUT /messages/1
{
  "text": "oh hai!"
}
```

instead of what is required by every other adapter (I'm pretty sure):

```json
// PUT /messages/1
{
  "text": "oh hai!",
  "id": 1
}
```

I'm ok with that. I just want to make sure we are on the same page because the other database adapters don't behave like that. For `PUT`'s, whatever you want to store needs to be passed in the request body (ie. `data`). For `PATCH` whatever you want to change or add is sent in the request body (ie. `data`).